ddtrace/tracer/transport.go

```diff
@@ -56,12 +55,13 @@ func defaultHTTPClient(timeout time.Duration) *http.Client {
 }
 
 const (
-	defaultHostname    = "localhost"
-	defaultPort        = "8126"
-	defaultAddress     = defaultHostname + ":" + defaultPort
-	defaultURL         = "http://" + defaultAddress
-	defaultHTTPTimeout = 10 * time.Second        // defines the current timeout before giving up with the send process
-	traceCountHeader   = "X-Datadog-Trace-Count" // header containing the number of traces in the payload
+	defaultHostname          = "localhost"
+	defaultPort              = "8126"
+	defaultAddress           = defaultHostname + ":" + defaultPort
+	defaultURL               = "http://" + defaultAddress
+	defaultHTTPTimeout       = 10 * time.Second              // defines the current timeout before giving up with the send process
+	traceCountHeader         = "X-Datadog-Trace-Count"       // header containing the number of traces in the payload
+	obfuscationVersionHeader = "Datadog-Obfuscation-Version" // header containing the version of obfuscation used, if any
```

Comment:
out of curiosity, why do some `Datadog-` headers have the `X-` prefix, but others don't? Should it be added here?

Reply:
I had to look this up, apparently the old MIME headers [RFC](https://www.ietf.org/rfc/rfc2047.txt) said this should be done for custom headers. However since at least 2019 that's apparently been deprecated and you can just name headers whatever you want :P https://datatracker.ietf.org/doc/html/draft-saintandre-xdash-00 I noticed some of our more recent headers dropped the `X-` so it seemed to make sense to do that here as well